packages/pds-ember/README.md

```diff
@@ -19,15 +19,47 @@ ember install @hashicorp/pds-ember
 
 # Usage
 
-[Longer description of how to use the addon in apps.]
+## Styles
+`ember-cli-sass` is required to make use of provided Sass assets.
```

Comment:
Is it worth noting that you have to use dart-sass? That's the default now, but could be important.

Reply:
Currently, `sass` is an alias, but I just saw that it's [deprecated](http://npmjs.com/package/sass) in favor of `dart-sass`.

I'll update.

Reply:
Actually, when you run `ember install ember-cli-sass` the addon handles installation of the appropriate Sass version, so there's nothing really to note, is there?

Reply:
Added note below command.